src/site/apt/examples/using-inline-descriptors.apt.vm

```diff
@@ -0,0 +1,86 @@
+ ------
+  Using Inline Assembly Descriptors
+ ------
+  Slawomir Jaranowski
+ ------
+  2023-12-17
+ ------
+
+~~ Licensed to the Apache Software Foundation (ASF) under one
+~~ or more contributor license agreements.  See the NOTICE file
+~~ distributed with this work for additional information
+~~ regarding copyright ownership.  The ASF licenses this file
+~~ to you under the Apache License, Version 2.0 (the
+~~ "License"); you may not use this file except in compliance
+~~ with the License.  You may obtain a copy of the License at
+~~
+~~   http://www.apache.org/licenses/LICENSE-2.0
+~~
+~~ Unless required by applicable law or agreed to in writing,
+~~ software distributed under the License is distributed on an
+~~ "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+~~ KIND, either express or implied.  See the License for the
+~~ specific language governing permissions and limitations
+~~ under the License.
+
+~~ NOTE: For help with the syntax of this file, see:
+~~ https://maven.apache.org/doxia/references/apt-format.html
+
+Using Inline Assembly Descriptors
+
+* Introduction
+
+  For simple usage we can inline Assembly descriptor into project configuration.
+  We don't need to create additional file with Assembly descriptor.
+
+  It can allow simplify configuration in case of inherited,
```

Comment:
delete "allow"
sounds like a sentence fragment? Is something missing? inherited what?